shell.nix

```diff
@@ -16,10 +16,10 @@ let
   ioSelfBuild = pkgs.lib.overrideDerivation iohk-ops.env (drv: {
     shellHook = ''
       function io {
-        runhaskell -iiohk iohk/iohk-ops.hs "$@"
+        cabal exec iohk-ops -- "$@"
       }
       function ghcid-io {
-        ${pkgs.haskellPackages.ghcid}/bin/ghcid -c "ghci -iiohk iohk/iohk-ops.hs"
+        ${pkgs.haskellPackages.ghcid}/bin/ghcid -c "ghci -iiohk -iiohk/common iohk/iohk-ops.hs"
```

Comment:
`ghcid -c "runhaskell Setup.hs repl"` again
although, there is the pain-point that repl has to be ran in the iohk dir, and the io util wants to not be ran there (but i suspect `-C ../config.yaml` can help)

Reply:
Unfortunately `-C ../config.yaml` doesn't work because it won't look up topology files relative to the config file. We should fix that, but not here.